package.json

```diff
@@ -14,7 +14,8 @@
         "yarn": "^1.22.10"
     },
     "workspaces": [
-        "keith-vscode"
+        "keith-vscode",
+        "table-webview"
```

Comment:
I think we already discussed this, but just in case. I don't think we want another workspace for the table view

Reply:
Can you repeat the reason for this?